.github/workflows/vuln-check-reusable.yaml

```diff
@@ -0,0 +1,151 @@
+name: Vulnerability Check
+
+on:
+  workflow_call:
+    inputs:
+      target-ref:
+        description: 'Target ref (branch, tag, release) to scan'
+        required: true
+        type: string
+        default: 'main'
+      find-latest-release:
+        description: 'Flag to find the latest version for specified `target-ref`'
+        required: false
+        type: boolean
+        default: false
+      images:
+        description: "JSON string representing a list of pairs of names and image base names of Docker images. For example, if `'[[\"ScalarDL Ledger\", \"scalar-ledger\"], [\"ScalarDL Auditor\", \"scalar-auditor\"]'` is specified, the scanner will scan `ghcr.io/scalar-labs/scalar-ledger:<tag>` and `ghcr.io/scalar-labs/scalar-auditor:<tag>`."
+        required: true
+        type: string
+      version-command:
+        description: 'Shell command that prints the version string. It is used as the tag for Docker images to be scanned.'
+        required: false
+        type: string
+        default: "./gradlew :common:properties -q | grep version: | awk '{print $2}'"
+    secrets:
+      CR_PAT:
+        required: true
+      SLACK_SECURITY_WEBHOOK_URL:
+        required: true
```

Comment:
Have you considered using the secrets `inherit` feature so that we don't need to specific secrets as required input of this reusable workflow ?
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idsecretsinherit

Reply:
Thank you for the comment! Good point. I have tried the `inherit` once. It can reduce the repeating code, especially in the [`scheduled-vuln-check.yaml`](https://github.com/scalar-labs/scalardb/blob/master/.github/workflows/scheduled-vuln-check.yaml). 
I found the downside is that it becomes difficult to know what secrets are needed from the "caller" workflow side. What do you think?

Reply:
Anyway, the change to use the `inherit` will be on the workflow "caller" repositories, so I don't think it will affect this PR.

Reply:
> I found the downside is that it becomes difficult to know what secrets are needed from the "caller" workflow side. 

I agree. Maybe mentioning what secrets are needed in the workflow in comments can clarify things?  Anyway, I don't have a strong opinion on it, so not using it is fine too.

If `inherit` is used by the caller workflow, I don't think you need to have this block 
```
secrets:
      CR_PAT:
        required: true
      SLACK_SECURITY_WEBHOOK_URL:
        required: true
```
but keeping it does cause an error maybe.

Reply:
> If inherit is used by the caller workflow, I don't think you need to have this block

Ah, I missed this point. Then, we can remove this block. But a little unsure if it is okay to remove the `required` flag together.
Let's keep this block for now as the original workflows do. We can improve them later.

Reply:
Ok, thanks.